### ISSUE TITLE
gbp

### ISSUE THREAD

**Issue by Shimmer2** [libretro/RetroArch#15928]
When using the game boy pocket shader, there are no gray gaps around the screen, which does not look like a real machine. When using the game boy pocket shader with borders, there are gray gaps that look very similar to a real machine, but I want to achieve the effect shown in Figure 2 when using the game boy pocket shader. What should I do?
<img width="800" alt="截屏2023-11-19 下午8 31 40" src="https://github.com/libretro/RetroArch/assets/79088619/faa6b3d3-c8e5-447a-8fa9-00f342785ddd">
<img width="800" alt="未命名" src="https://github.com/libretro/RetroArch/assets/79088619/1ec2aced-1da9-480d-a531-c6780e98407c">

**Comment by hizzlekizzle** [comment]
Hmm, I don't have the problem. The gameboy-pocket shader from the 'shaders_slang/handheld' directory looks like this for me:
![image](https://github.com/libretro/RetroArch/assets/2805586/926fcbfc-f83f-4f9f-8c9e-ed2de036460f)

**Comment by Shimmer2** [comment]
Thank you very much for your reply! It has been resolved, as long as the integer magnification with a resolution greater than 160x144 is set, the border will appear. Another question I need to ask is, can override directly configure shaders? I tried to do this but failed

**Comment by hizzlekizzle** [comment]
No, overrides don't include shader stuff, other than whether shaders are enabled at all or not. You can save shader configurations in the 'save' submenu of the shader menu.

**Comment by Shimmer2** [comment]
Retroarch comes with only black gbp masks, but I would like other colors, such as blue. Where can I download them? I searched for a long time but couldn't find it






------------------ Original ------------------
From: hizzlekizzle ***@***.***&gt;
Date: Mon,Nov 20,2023 3:31 AM
To: libretro/RetroArch ***@***.***&gt;
Cc: Shimmer2 ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [libretro/RetroArch] gbp (Issue #15928)





 
No, overrides don't include shader stuff, other than whether shaders are enabled at all or not. You can save shader configurations in the 'save' submenu of the shader menu.

**Comment by Shimmer2** [comment]
I am a Chinese user, thank you very much for the Retroarch developed by your team! I want to support your hard work! But Alipay or WeChat payment is popular in China. I wonder if it can be supported later?







------------------ Original ------------------
From: hizzlekizzle ***@***.***&gt;
Date: Mon,Nov 20,2023 3:31 AM
To: libretro/RetroArch ***@***.***&gt;
Cc: Shimmer2 ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [libretro/RetroArch] gbp (Issue #15928)





 
No, overrides don't include shader stuff, other than whether shaders are enabled at all or not. You can save shader configurations in the 'save' submenu of the shader menu.

**Comment by hizzlekizzle** [comment]
You can get a bunch more from this thread on the forum: https://forums.libretro.com/t/handheld-border-shaders/2551

As for support, we may be able to support one or both of those at some point, but in the meantime, we're just glad you're enjoying the program. Thanks for the kind words :)

**Comment by Shimmer2** [comment]
The maximum zoom of the mask border can only be 2 times. Is there any way to support a larger magnification?
